CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# How to contribute to DelugeFirmware/community
+
+Above all, thank you for your interest in both the Synthstrom Deluge and being a contributor!
+
+Please see also, the [Code of Conduct](CODE_OF_CONDUCT.md) document.
+
+## Bug reports
+
+* Please submit an issue highlighting as much detail and context about the bug as possible
+
+* Provide detail on whether the bug was found on an OLED or 7seg Deluge
```

Comment:
Yes!

Not trying to derail this PR at all, but we could set up issue templates so that folks filing bugs are immediately presented with empty fields to populate. It's been super useful or my day-job, at least, at keeping the description quality high.